server/src/main/java/org/elasticsearch/node/NodeConstruction.java

```diff
@@ -258,7 +258,7 @@ static NodeConstruction prepareConstruction(
     /**
      * See comments on Node#logger for why this is not static
      */
-    private final Logger logger = LogManager.getLogger(Node.class);
```

Comment:
This should not be static

Reply:
🤦 sorry about that, fixed all 3 spots you pointed out :)